fontdrasil/src/types.rs

```diff
@@ -10,6 +10,9 @@ use smol_str::SmolStr;
 #[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
 pub struct GlyphName(SmolStr);
 
+/// The name of the undefined glyph
+pub static NOTDEF: GlyphName = GlyphName(SmolStr::new_inline(".notdef"));
```

Comment:
A common pattern here is to define the const in the impl of the type itself, like,

```rust
impl GlyphName {
    pub static NOTDEF: GlyphName = ...;
}
```

and then you can reference it anywhere that `GlyphName` is in scope, as `GlyphName::NOTDEF`.

Reply:
Much nicer, updated so.